website/docs/reference/widgets/table/inline-editing.md

```diff
@@ -208,29 +251,30 @@ This property contains all the details of the edited rows. It has the following
 
 #### updatedRowIndices
 
-This binding property displays the index number of the updated row. It contains an array of edited row indices.&#x20;
+This binding property displays the index number of the updated row. It contains an array of edited row indices.
 
-For example, if you updated the second row of a table. Now, when you bind this property into a text widget, the property displays the outcome as follows:&#x20;
+For example, if you update the second and fourth rows of a table, the `updatedRowIndices` property contains the value:
 
-```
+```javascript
 [ 
-1
+  1,
+  3
 ]
 ```
 
 #### updatedRow
 
-This property consists of the details of the row that recently got updated. Irrespective of the update mode, the `updatedRow` property consists of the row that got recently updated.
+This property contains the details of the row that was recently updated, regardless of the **Update mode**. For example, if you bind this property into a text widget, you get an output something like this:
```

Comment:
Can we add that this is useful for single row update mode?

Reply:
and @spark-c maybe we should move it where we are explaining Single Row Mode. That way we are also visually segregating them.

<img width="1064" alt="Screenshot 2022-12-12 at 18 13 38" src="https://user-images.githubusercontent.com/32227135/207048250-6fb2be69-ef1a-43dd-9c78-85152d8ff042.png">

Reply:
@jnikhila is it ok update the CSS of the headings?

Reply:
Yes, we discussed it in one of our standup calls for this. @spark-c - As discussed earlier, have we taken into account the font-size ticket #590  that's already in the backlog, and will the CSS changes resolve that too?